sale_elaboration/views/sale_elaboration_report_views.xml

```diff
@@ -9,7 +9,7 @@
         <field name="model">sale.order.line</field>
         <field name="arch" type="xml">
             <tree string="Elaborations" create="false">
-                <field name="confirmation_date"/>
+                <field name="date_order"/>
```

Comment:
Why are you using order date now instead of the confirmation date?

Reply:
Un V13 the confirmation_date order field is deleted, now confirmation date is date_order if order is confirmed